### PR TITLE
Removed ipfix references from automation

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Resolves [NETOBSERV-528](https://issues.redhat.com/browse/NETOBSERV-528)

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -237,8 +232,7 @@ pipeline {
                     // attempt updating common parameters of flowcollector
                     println 'Updating common parameters of flowcollector...'
                     returnCode = sh(returnStatus: true, script: """
-                        oc patch flowcollector cluster --type=json -p "[{"op": "replace", "path": "/spec/agent", "value": ${params.COLLECTOR_AGENT}}] -n network-observability"
```
Comment (memodi) [inline Jenkinsfile:240]: we would need this patch until upstream [flowcollector CRD](https://raw.githubusercontent.com/netobserv/network-observability-operator/main/config/samples/flows_v1alpha1_flowcollector_versioned.yaml) is updated which is used [here](https://github.com/openshift-qe/ocp-qe-perfscale-ci/blob/523cac2c9bd62f9c44bde3cfa60c7342a6b2faaf/scripts/netobserv.sh#L12 )

**Review by memodi — APPROVED** [review]

**Review by memodi — CHANGES_REQUESTED** [review]
requesting changes to restore patch, could use hardcoded ebpf agent in there.

File: Jenkinsfile (hunk 1 above)
Comment (nathan-weinberg) [reply Jenkinsfile:240]: Done

**Pull request merged**